bin/verilator

```diff
@@ -419,17 +419,20 @@ detailed descriptions of these arguments.
     --vpi                       Enable VPI compiles
     --waiver-output <filename>  Create a waiver file based on the linter warnings
      -Wall                      Enable all style warnings
+     -Werror-UNUSED             Convert unused genvar, param and signal warnings to errors
```

Comment:
I think this is a special case of -Werror-<message> so UNUSED doesn't need to be listed in this file.  Then where warnings are listed just have UNUSED described as it controlling all three.

Reply:
I removed the warning there and added it to docs/guide/warnings.rst file